### ISSUE TITLE
not an issue, a feature request

### ISSUE THREAD

**Issue by Benjaneer** [Xbozon/storyteller#36]
Now that you've converted it to a sheet style (or whatever the correct nomenclature is), which is awesome by the way, could you make the book and newspaper two different styles so that some entries can be one and other entries can be the other (instead on switching all of them in the settings, effectively restricting us to only one)?

**Comment by Xbozon** [comment]
@Benjaneer 

Yes, of course. That was in the plans until I had a mountain of things to study. I decided to release the update in a minimal form. Thanks for using the mod and offering ideas, I'll try to come back soon!

**Comment by Benjaneer** [comment]
I figured out how to do it in my local files and tried to upload it into git (I've written allot of code but haven't used git much, sorry if it hit you with a bunch of messages).

**Comment by Benjaneer** [comment]
nevermind :)
https://github.com/Benjaneer/storyteller-addon-singlesheet
